desk/app/channels.hoon

```diff
@@ -767,11 +767,18 @@
   ::
   ++  ca-has-sub
     ^-  ?
-    (~(has by wex.bowl) [ca-sub-wire ship.nest dap.bowl])
+    (~(has by wex.bowl) [ca-sub-wire ship.nest %channels-server])
```

Comment:
Should probably say `server` because that's what `+ca-safe-sub` uses.

```suggestion
    (~(has by wex.bowl) [ca-sub-wire ship.nest server])
```

("They should all be `%channels-server`" is a valid response, but also out of scope here.)